receiver/gitproviderreceiver/README.md

```diff
@@ -87,6 +87,19 @@ service:
             exporters: [...]
 ```
 
+### Github PAT basic permissions
+* repo:All
```

Comment:
Shouldn't it just be `repo:read` or `read:repo`? There's nothing in here that writes to GitHub as all calls are read only.